fastly-rails.gemspec

```diff
@@ -14,9 +14,11 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.rdoc"]
   s.test_files = Dir["test/**/*"]
 
-  s.add_dependency "rails"
+  s.add_dependency "railties"
   s.add_dependency 'fastly', '~> 1.2.1'
 
+  s.add_runtime_dependency('mime-types', ['>= 1.16', '< 3'])
```

Comment:
@clupprich can you comment on the version restriction here? Mostly curious how you arrived at these versions.